### DISCUSSION TITLE
[CHORE] How to set the dynamic expression for Fork (flow) activity branches?

### DISCUSSION THREAD

**Discussion by shxylgithub** [elsa-workflows/elsa-core#5789] [Q&A]
How to set the dynamic expression for Fork (flow) activity branches?

**Comment by sfmskywalker** [comment]
The Fork activity's branches do not have dynamic expressions. When a Fork activity executes, all of its branches will execute next. The purpose of Fork is to split into multiple paths that execute in parallel.

If you need to split into multiple branches conditionally, you might want to use the Switch activity instead. The Switch activity is similar to Fork, but it allows you to provide a condition for each case (which is the functional equivalent of a branch in a fork).
